ids/mapping.json

```diff
@@ -6,6 +6,7 @@
   "manjot": "manjots1607",
   "nikhil": "whydonti",
   "om": "strider99",
+  "praveen":"praveen",
```

Comment:
"RDS-id" : "github-id"

Please add your github id in there.

Reply:
Corrected now.